spharpy/special.py

```diff
@@ -159,6 +159,8 @@ def _spherical_hankel_derivative(n, z, kind):
 
 def spherical_harmonic(n, m, theta, phi):
     """The spherical harmonics of order n and degree m.
+    The spherical harmonic functions are fully normalized (N3D) and
```

Comment:
```suggestion

    The spherical harmonic functions are fully normalized (N3D) and
```
very picky suggestion^^